config_examples/config_okex_futures.example.json

```diff
@@ -0,0 +1,93 @@
+{
+    "max_open_trades": 3,
+    "stake_currency": "USDT",
+    "stake_amount": 0.05,
+    "tradable_balance_ratio": 0.99,
+    "fiat_display_currency": "USD",
+    "timeframe": "5m",
+    "dry_run": true,
+    "cancel_open_orders_on_exit": false,
+    "trading_mode": "futures",
+    "margin_mode": "isolated",
+    "unfilledtimeout": {
+        "buy": 10,
+        "sell": 10,
+        "exit_timeout_count": 0,
+        "unit": "minutes"
+    },
+    "bid_strategy": {
+        "ask_last_balance": 0.0,
+        "use_order_book": true,
+        "order_book_top": 1,
+        "check_depth_of_market": {
+            "enabled": false,
+            "bids_to_ask_delta": 1
+        }
+    },
+    "ask_strategy": {
+        "use_order_book": true,
+        "order_book_top": 1
+    },
+    "exchange": {
+        "name": "okex",
+        "key": "your_exchange_key",
+        "secret": "your_exchange_secret",
+        "ccxt_config": {},
+        "ccxt_async_config": {
+        },
+        "pair_whitelist": [
+            "BTC/USDT:USDT",
+            "ETH/USDT:USDT",
+            "BAT/USDT:USDT",
+            "BCH/USDT:USDT",
+            "EOS/USDT:USDT",
+            "IOTA/USDT:USDT",
+            "LINK/USDT:USDT",
+            "LTC/USDT:USDT",
+            "NEO/USDT:USDT",
+            "XMR/USDT:USDT",
+            "XRP/USDT:USDT",
+            "XTZ/USDT:USDT"
```

Comment:
I thought this might be useful just for the pairlist

Reply:
when i commented the above, this was not the pairlist in this file (which is why i said - it's wrong).

that said - i don't think there's much benefit to it (not enough to justify more sample files we fail to maintain properly, anyway ...).
you can get the same pairlist with `freqtrade list-pairs --exchange okex --trading-mode futures` (which will actually list ALL pairs - which a sample config will never be able to).